networking/configuring-ptp.adoc

```diff
@@ -11,19 +11,42 @@ include::modules/technology-preview.adoc[leveloffset=+1]
 [id="about-using-ptp-hardware"]
 == About PTP hardware
 
-{product-title} includes the capability to use Precision Time Protocol (PTP)hardware on your nodes.
+{product-title} includes the capability to use Precision Time Protocol (PTP) hardware on your nodes.
 You can configure linuxptp services on nodes in your cluster that have PTP-capable hardware.
 
 [NOTE]
 ====
-The PTP Operator works with PTP-capable devices on clusters provisioned only on bare metal infrastructure.
+The PTP Operator works with PTP-capable devices on clusters provisioned only on bare-metal infrastructure.
 ====
 
 You can use the {product-title} console to install PTP by deploying the PTP Operator. The PTP Operator creates and manages the linuxptp services. The Operator provides the following features:
 
 * Discovery of the PTP-capable devices in a cluster.
 * Management of the configuration of linuxptp services.
+* Notification of PTP clock events that negatively affect the performance and reliability of your application with the fast events PTP Operator sidecar.
+
+include::networking/installing-ptp-operator.adoc[leveloffset=+1]
```

Comment:
@vikram-redhat, are we allowing assemblies included in other assemblies now?

Reply:
Actually not sure this assembly is serving any purpose. I'll remove it.